### ISSUE TITLE
Colour contrast failure on placeholder text in the search field in the header

### ISSUE THREAD

**Issue by mcheung-nhs** [nhsuk/nhsuk-frontend#687]
## What is the issue?
The colour of the placeholder text in the search field in the header does not meet the minimum colour contrast ratio 4.5:1 for small text (WCAG 1.4.3) 

* Device: Macbook Pro, Surface Pro 4
* Operating System: MacOS, WIndows
* Browser: Firefox, Chrome, Safari, Edge
* Browser version: latest
* NHS.UK frontend package version: current

## Is there anything else you think would be useful in recreating the issue?
The default placeholder text for Firefox seems lighter than that on Chrome or Safari or Edge

**Comment by davidhunter08** [comment]
Changing it from `$color_nhsuk-grey-2` to `$color_nhsuk-grey-1` will fix it.
